yci/skills/blast-radius/references/error-messages.md

```
@@ -0,0 +1,378 @@
+### Skill orchestration (`SKILL.md` flag parsing and output routing)
+
+---
+
+### `br-missing-change-file`
+
+- **ID**: `br-missing-change-file`
+- **Producer**: skill orchestration
+- **Exit code**: 1
+- **Trigger**: `--change-file` flag was not provided when invoking the skill.
+- **Message**:
+  ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)